tests/testflows/rbac/requirements/requirements.md

```diff
@@ -3186,14 +3226,14 @@ on tables created using the following engines
 * ReplicatedVersionedCollapsingMergeTree
 * ReplicatedGraphiteMergeTree
 
-##### AlterIndex
+##### Alter Index
 
 ###### RQ.SRS-006.RBAC.Privileges.AlterIndex
 version: 1.0
 
 [ClickHouse] SHALL support controlling access to the **alter index** privilege
 for a database or a specific table to one or more **users** or **roles**.
-Any `ALTER TABLE ... ORDER BY | ADD|DROP|MATERIALIZE|CLEAR INDEX` statements SHALL
+Any `ALTER TABLE ... ORDER BY | ADD|CREATE|MATERIALIZE|CLEAR INDEX` statements SHALL
```

Comment:
We don't have `ALTER TABLE ... CREATE INDEX` command, but we have `ALTER TABLE ... DROP INDEX` command.

Reply:
@MyroTk